### PR TITLE
models

### PR DESCRIPTION
**Pull request by ashkhurana**
Hi all, 

I have updated the models. A few things to check

Added validation for answer content - length (maxim 100 ) - We may want to change it to 150??
same for question and user models

One thing I have noticed we don't have a bio/about me for university - Is this something we want to add?

### CODE REVIEW

File: app/models/answer.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 class Answer < ApplicationRecord
   belongs_to :user
   belongs_to :question
+
+  validates :user, presence: true
+  validates :question, presence: true
+  validates :content, length: { maximum: 100 }
```
Comment (ConnorSand) [inline app/models/answer.rb:7]: I think removing this restriction to allow the default maximum size for type 'text' (30,000 characters) is okay. For reference, the maximum number of characters allowed for type 'string' is 255. 
There may be a situation where university students ask questions that warrant a lengthy response for subjective questions such as an English analysis?

Comment (ConnorSand) [inline app/models/answer.rb]: As you mentioned I think the maximum lengths on the content fields can be increased to possible ~300 or 400. There may be some subject questions that are subjective in nature that warrant a longer response?

File: app/models/question.rb [hunk 2]
```diff
@@ -1,3 +1,7 @@
 class Question < ApplicationRecord
   belongs_to :user
+  has_many :answers
+
+  validates :user, presence: true
+  validates :content, length: { maximum: 50 }
```
Comment (ConnorSand) [inline app/models/question.rb]: Limiting the question makes sense as we want concise questions. Maybe we could make this ~100 to be safe tho?

Comment (ashkhurana) [reply app/models/question.rb]: sounds good, let me update

File: app/models/answer.rb (hunk 1 above)
Comment (ashkhurana) [reply app/models/answer.rb]: sounds good, let me update

**Pull request merged**